lib/ui/src/modules/ui/components/left_panel/text_filter.js

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import debounce from 'lodash.debounce';
+import { debounce } from 'lodash';
```

Comment:
Yeah, that what I did.. But problem looks like in a way of the import.. In the transpiled version 
 of the `import ... from lodash.debounce` I am getting a reference to the `debounce` function. And in transpiled version of named import - the reference is to the `lodash` object, in this case I managed to replace its method. :face_with_head_bandage: